app/src/main/java/com/limelight/computers/ComputerDatabaseManager.java

```diff
@@ -9,7 +9,9 @@
 import java.util.List;
```

Comment:
@kmreisi  Again it doesn't work.
I opened the app

![issue 1](https://user-images.githubusercontent.com/5093683/213966532-7f059492-7a7f-42fd-a210-95fbb7d8a7a1.jpg)


I had waited for about 10 seconds and this was what I got

![issue 2](https://user-images.githubusercontent.com/5093683/213966539-09626330-4ac6-420e-b804-d8a79ce99468.jpg)